src/dbSta/src/CMakeLists.txt

```diff
@@ -66,9 +86,6 @@ swig_lib(NAME          dbSta
 target_sources(dbSta
   PRIVATE
     dbSta.cc
```

Comment:
This file contains things that need to be in the library.  I think you'll have to split it up.

Reply:
I hope you address this in a future PR.  It doesn't matter in the current OR app